python/cuml/dask/extended/linear_model/logistic_regression.py

```diff
@@ -0,0 +1,210 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.dask.common.base import BaseEstimator
+from cuml.common import with_cupy_rmm
+from cuml.common.import_utils import has_daskglm
+
+import cupy as cp
+import numpy as np
+from dask.utils import is_dataframe_like, is_series_like, is_arraylike
+import cudf
+
+
+class LogisticRegression(BaseEstimator):
+    """
+    Distributed Logistic Regression for Binary classification.
+
+
+    Parameters
+    ----------
+    fit_intercept: boolean (default = True)
+       If True, the model tries to correct for the global mean of y.
+       If False, the model expects that you have centered the data.
+    solver : 'admm'
+        Solver to use. Only admm is supported currently.
+    penalty : {'l1', 'l2'} (default = 'l2')
+        Regularization technique for the solver.
+    C: float (default = 1.0)
+       Inverse of regularization strength; must be a positive float.
+    max_iter: int (default = 100)
```

Comment:
Just curious why `100` in particular

Reply:
This is the default in dask-glm, which I suspect is the default because it's the default in scikit-learn's LogisticRegression class https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html